components/workflow.rst

```diff
@@ -83,7 +83,7 @@ you can retreive a workflow from it and use it as follows::
 
     // ...
     // Consider that $blogPost is in place "draft" by default
-    // Consider that $post is in state "draft" by default
+    // Consider that $blogPost is in state "draft" by default
```

Comment:
The line is duplicated. Maybe it will be fixed after rebasing and fixing the conflicts.